cmake/usd.cmake

```diff
@@ -15,9 +15,15 @@
 #
 function(init_usd)
     # Adjust PYTHONPATH, PATH
-    mayaUsd_append_path_to_env_var("PYTHONPATH" "${PXR_USD_LOCATION}/lib/python")
+    if (DEFINED MAYAUSD_TO_USD_RELATIVE_PATH)
+        set(USD_INSTALL_LOCATION "${CMAKE_INSTALL_PREFIX}/${MAYAUSD_TO_USD_RELATIVE_PATH}")
+    else()
+        set(USD_INSTALL_LOCATION ${PXR_USD_LOCATION})
+    endif()
```

Comment:
Again when given a USD relative path, use that for the PATH and PYTHONPATH.

Reply:
I realized (from a clean build) that this change breaks the build. We need the PATH and PYTHONPATH to usd during the configure step for generating the schemas. So for that step we need to use USD from it's original location.

But then for running the tests we would want PATH & PYTHONPATH to point to the USD in the install folder.  I need to think some more how to fix this.